rmw_implementation/CMakeLists.txt

```diff
@@ -38,17 +38,30 @@ message(STATUS "")
 message(STATUS "Using default rmw implementation: '${RMW_IMPLEMENTATION}'")
 message(STATUS "")
 
-# if only a single rmw impl. is available or poco is not available
-# this package will directly reference the default rmw impl.
-if(NOT RMW_IMPLEMENTATIONS MATCHES ";" OR NOT Poco_FOUND)
-  set(RMW_IMPLEMENTATION_SUPPORTS_POCO FALSE)
+set(RMW_IMPLEMENTATION_FORCE_POCO OFF CACHE BOOL
+    "Force using poco for loading shared libraries, even if there is only one \
+     implementation available")
 
-else()
+set(RMW_IMPLEMENTATION_SUPPORTS_POCO FALSE)
+# if poco is not forced and only a single rmw impl. is available or poco is not available
+# this package will directly reference the default rmw impl.
+if(RMW_IMPLEMENTATION_FORCE_POCO OR (RMW_IMPLEMENTATIONS MATCHES ";" AND POCO_FOUND))
```

Comment:
Use consistent case of CMake variables: `Poco_FOUND`.

Reply:
Done - oops!